CODEOWNERS

```diff
@@ -11,7 +11,9 @@
 # include files
 src/include/sof/dmic.h			@singalsu
 src/include/host/*			@ranj063
-src/include/uapi/**			@thesofproject/steering-committee
+src/include/ipc/**			@thesofproject/steering-committee
+src/include/kernel/**			@thesofproject/steering-committee
+src/include/user/**			@thesofproject/steering-committee
```

Comment:
Question: what does the "**" do here? Is this better or different than a single "*"?

Reply:
`*` matches only one level of files. If you want to match whole subdir tree you have to use `**`